components/SideBar.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+  <v-navigation-drawer
+    v-model="drawer"
+    :width="50"
+    class="mt-10 ma-2"
+    color="#FFFFFF00"
```

Comment:
what is this color, name it once and use it in the entire app

Reply:
I will try to fix it later, it's transparent